packages/server/test/unit/socket_spec.js

```diff
@@ -122,287 +121,6 @@ describe('lib/socket', () => {
       })
     })
 
-    context('on(automation:request)', () => {
```

Comment:
Some of these tests I wanted to keep but I can't even get them to run locally ☹️